app/components/forms/song/StanzaFields/StanzaFields.tsx

```diff
@@ -76,14 +84,15 @@ export const StanzaFields: React.FC<{
             <Fieldset.Headless namespace={`lines[${index}]`}>
               <LineFields
                 line={line}
+                updateLine={updateLine(index)}
                 deleteLine={deleteLine(index)}
                 insertLine={insertLine(index)}
               />
             </Fieldset.Headless>
             <button
               type="button"
               aria-label="Add Line"
-              onClick={insertLine(index)}
+              onClick={() => insertLine(index)()}
```

Comment:
This is gross and I will refactor it later lol, but because we added a parameter, this is what we have to do or else the `e` event with be passed in as the lyrics.